packages/react-scripts/template/src/registerServiceWorker.js

```diff
@@ -7,6 +7,32 @@
 
 // To learn more about the benefits of this model, read https://goo.gl/KwvDNy.
 // This link also includes instructions on opting out of this behavior.
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Toast from './Toast';
+
+const useToast = true;
+const domId = 'toast';
+let dom = document.getElementById(domId);
+
+if (!dom) {
```

Comment:
If this is suppose to be once off on load. Maybe we should remove the created dom node after the timeout ends / it's dismissed.

Don't think its necessary but could be nice to avoid any possibility of conflicting styles.